credential/status/statuslist2021.go

```diff
@@ -0,0 +1,213 @@
+package status
```

Comment:
work is here

Reply:
@thehenrytsai sorry about the unrelated changes, this file is the implementation of the specification